osu.Game.Tests/Visual/Gameplay/TestScenePlayerLoader.cs

```diff
@@ -68,8 +69,22 @@ public void TestBlockLoadViaMouseMovement()
         {
             AddStep("load dummy beatmap", () => ResetPlayer(false));
             AddUntilStep("wait for current", () => loader.IsCurrentScreen());
-            AddRepeatStep("move mouse", () => InputManager.MoveMouseTo(loader.VisualSettings.ScreenSpaceDrawQuad.TopLeft + (loader.VisualSettings.ScreenSpaceDrawQuad.BottomRight - loader.VisualSettings.ScreenSpaceDrawQuad.TopLeft) * RNG.NextSingle()), 20);
+            AddRepeatStep("move mouse on settings", () => InputManager.MoveMouseTo(loader.VisualSettings.ScreenSpaceDrawQuad.TopLeft + (loader.VisualSettings.ScreenSpaceDrawQuad.BottomRight - loader.VisualSettings.ScreenSpaceDrawQuad.TopLeft) * RNG.NextSingle()), 20);
             AddAssert("loader still active", () => loader.IsCurrentScreen());
+            AddRepeatStep("move mouse on loader", () => InputManager.MoveMouseTo(loader.ScreenSpaceDrawQuad.TopLeft + (new Vector2(loader.VisualSettings.ScreenSpaceDrawQuad.TopLeft.X, loader.ScreenSpaceDrawQuad.BottomLeft.Y) - loader.ScreenSpaceDrawQuad.TopLeft) * RNG.NextSingle()), 20);
```

Comment:
this is testing the opposite of what we want. this should still load.

Reply:
moving mouse on loader repeatedly is what we call not idle, it should be ready to push only when the user is idle?

Reply:
the idea is we *only want to block* if the user is doing something meaningful. check on master – it blocks if they are adjusting settings only. your change means if someone is playing with their mouse it will never load the beatmap.